statsmodels/genmod/families/varfuncs.py

```diff
@@ -45,14 +45,11 @@ def __call__(self, mu):
         mu = np.asarray(mu)
         return np.ones(mu.shape, np.float64)
 
-
     def deriv(self, mu):
         """
         Derivative of the variance function v'(mu)
         """
-        from statsmodels.tools.numdiff import approx_fprime_cs
-        # TODO: diag workaround proplem with numdiff for 1d
-        return np.diag(approx_fprime_cs(mu, self))
+        return np.zeros_like(mu)
```

Comment:
There is a problem now with inheriting from `VarianceFunction`
We need to decide whether it is a base class that can be used for inheritance or a default constant variance class.

I think the only example for user defined variance function is in my risk ratio experiments with Poisson that has almost Binomial variance.